Core/scripts/dirac-install.py

```diff
@@ -2390,6 +2387,20 @@ def installDiracOS(releaseConfig):
   return True
 
 
+def installDiracOSPython3(releaseConfig):
+  """
+  Install DIRAC OS for Python 3.
+
+  :param ReleaseConfig releaseConfig: The ReleaseConfig object for configuring the installation
+  """
+  url = "https://github.com/chrisburr/DIRACOS2/releases/latest/download/DIRACOS-Linux-x86_64.sh"
```

Comment:
Just reminder for changing this link.

Reply:
This should be done as part of #4775. I haven't set up `DIRACGrid/DIRACOS2` yet as I'd like to have tags for the tornado packages rather than building from a branch.